src/index.js

```diff
@@ -1,22 +1,47 @@
-const plugin = require('tailwindcss/plugin')
+const plugin = require("tailwindcss/plugin");
 
-module.exports = plugin(function({ addUtilities, e }) {
-  addUtilities({
-    [`.${e('?')}`]: {
-      'outline-style': 'solid',
-      'animation': `${e('?')}wobble 0.6s ease-in-out alternate infinite`
-    },
-    [`@keyframes ${e('?')}wobble`]: {
-      '0%': {
-        'outline-width': '4px',
-        'outline-color': '#f16bc9',
-        'box-shadow': 'inset 4px 4px #f16bc9, inset -4px -4px #f16bc9'
-      },
-      '100%': {
-        'outline-width': '6px',
-        'outline-color': '#f71fb6',
-        'box-shadow': 'inset 6px 6px #f71fb6, inset -6px -6px #f71fb6'
-      },
-    }
-  });
-});
+module.exports = plugin.withOptions(
+  ({
+    animationDuration = "0.6s",
+    enableAnimation = true,
+    highlightColorStart = "#f16bc9",
+    highlightColorEnd = "#f71fb6",
+    widthStart = "8px",
+    widthEnd = "12px",
+  } = {}) => {
+    return function ({ addUtilities, e }) {
+      const BOX_SHADOW_INSET_START = "4px";
+      const BOX_SHADOW_INSET_END = "6px";
```

Comment:
So that the inset and outline are symmetrical, perhaps we should use widthStart and widthEnd for both?

If widthStart is 8px, we could apply 4px to both the inset and the outline. We'd need to do a little parsing of the string, but it might make for a nice api

What do you think?

Reply:
Something like this?

```
function parsePx(input, defaultValue) {
  let value = input.match(/^\d+px$/);
  if (value) {
    return parseInt(value[0], 10);
  }
  return defaultValue;
}
```

Reply:
Absolutely, that's a great idea. Small nitpick, but perhaps the regex `/\d+px/` would better approximate browser behavior when parsing the CSS value (i.e. ignoring whitespaces or chars before or after the `px` substring). Would it be worth it to sanitize this far? How does that sound?

Reply:
Sounds good, thanks